README.md

```diff
@@ -1,25 +1,35 @@
-[![Build status](https://travis-ci.org/rust-embedded/r0.svg?branch=master)](https://travis-ci.org/rust-embedded/r0)
+# r0
+
+[![Build status](https://api.travis-ci.org/rust-embedded/r0.svg?branch=master)](https://travis-ci.org/rust-embedded/r0)
 [![crates.io](https://img.shields.io/crates/d/r0.svg)](https://crates.io/crates/r0)
 [![crates.io](https://img.shields.io/crates/v/r0.svg)](https://crates.io/crates/r0)
 
-# `r0`
+Memory initialization code **crt0** written in Rust. This crate is for bare metal systems where there is no ELF loader or OS to take care of initializing RAM for the program. Is also provides an API to add «life before main» functionality to bare metal systems.
+
+This project is used by other crates, like:
+
+* [cortex-m-rt](https://github.com/rust-embedded/cortex-m-rt)
+* [riscv-rt](https://github.com/rust-embedded/riscv-rt)
+* [msp430-rt](https://github.com/rust-embedded/msp430-rt)
 
-Memory initialization code ("[crt0]") written in Rust.
+Wiki quote:
+
+> crt0 (also known as c0) is a set of execution startup routines linked into a C program that performs any initialization work required before calling the program's main function.
+> It generally takes the form of an object file called crt0.o, often written in assembly language, which is automatically included by the linker into every executable file it builds.
+
+Read more about [crt0](https://en.wikipedia.org/wiki/Crt0).
```

Comment:
The wiki quote also makes it sound like this is a crt0 implementation. I'd prefer to just link there, maybe with an explanation like

    This crate provides similar functionality to [crt0](https://en.wikipedia.org/wiki/Crt0) in the C runtime.

Reply:
Done